http_client.rst

```diff
@@ -318,12 +467,146 @@ called when new data is uploaded or downloaded and at least once per second::
 Any exceptions thrown from the callback will be wrapped in an instance of
 ``TransportExceptionInterface`` and will abort the request.
 
-Advanced Options
-~~~~~~~~~~~~~~~~
+Configuration
+~~~~~~~~~~~~~
+
+Check out the full :ref:`http_client config reference <reference-http-client>`
+to learn about all the options you might need to take full control of the way
+the request is performed, including DNS pre-resolution, SSL parameters, public
+key pinning, etc.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/framework.yaml
+        framework:
+            http_client:
+                max_host_connections: 10
+                default_options:
```

Comment:
This is the first time `default_options` appears in the article? Because we don't explain it. Why "options" and "default options"?